src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -918,6 +918,17 @@ void testCaseProgressReporter(bool *quit)
     }
 }
 
+void set_shell_env(std::string var_name, std::string trailing_path, int overwrite)
```

Comment:
this only works for "path"-like env, so probably call it set_shell_path_env()?

Reply:
Never pass std::string by value unless you truly need a copy.